database/migrations/2021_08_03_052622_create_channels_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateChannelsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('channels', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->mediumText('description')->nullable();
+            $table->timestamps();
+        });
+    }
+
```

Comment:
Can you also add a `slug` field? so that would be like: `$table->string('slug');`